docs/workforce-management-framework/topics/con-logging.adoc

```diff
@@ -0,0 +1,15 @@
+[id='logging-module-{chapter}']
+= Logging Module
+
+The {WFM-RC-NameLong} Logger Module is a logging facade for Node.js based applications and it is used by all {WFM-RC-NameLong} modules.
```

Comment:
Content here looks good, but it's great to have it not as "logger-module" but maybe something like
"Configuring and enabling logging for RainCatcher modules"? That's more user friendly and covers developer story rather than elements we have

Reply:
Proposal:
We could introduce logging at this level:
[
2. Introducing RainCatcher Features and Modules
2.1. RainCatcher Features
2.1.1. Introducing RainCatcher Security Features
**2.1.2. Introducing RainCatcher Logging**
]

and include a procedure at this level
[
4. Programming a RainCatcher Project
4.1. Working With PassportAuth
**4.2 Configuring Logging** 
]

WDYT?

Reply:
Build into PR (https://github.com/austincunningham/raincatcher-docs/pull/5/files)